src/modulefinder/sentry_modulefinder_linux.c

```diff
@@ -12,6 +12,9 @@
 #include <string.h>
 #include <unistd.h>
 
+#define MAX(a, b) ((a) > (b) ? (a) : (b))
```

Comment:
_nit_: Works in this case, but note that this would evaluate the parameters twice if they were function calls.